### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by TC1977**
## Description

- Updates the main README.md file by moving the "Additional Documentation" links to the docs/index.md file, where they were supposed to be.
- Updates index.md to also include other files that were previously added but not linked to anywhere (e.g. the Docker and cloud-init docs)
- Updates the Ansible roles to reflect the moves to Ansible 2.5 last summer and then 2.7.10 last month.
- Updates the SSH access instructions and success message to reflect current behavior.

## Motivation and Context

The docs haven't been updated as quickly as the code.

## How Has This Been Tested?

I clicked on the links.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [] I have read the **CONTRIBUTING** document.
- [] My code follows the code style of this project.
- [] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [] I have added tests to cover my changes.
- [] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,16 +166,14 @@ Use the example command below to start an SSH tunnel by replacing `user` and `ip
...
-To SSH into the Algo server for administrative purposes you can use the example command below by replacing `ip` with your own:
+To SSH into the Algo server for administrative purposes you can log into Terminal, go to the algo directory, and then use the command listed on the success message:
 
- `ssh root@ip -i ~/.ssh/algo.pem`
+ `ssh -i configs/algo.pem user@ip`
 
-If you find yourself regularly logging into Algo then it will be useful to load your Algo ssh key automatically. Add the following snippet to the bottom of `~/.bash_profile` to add it to your shell environment permanently.
+where `user` is either `root` or `ubuntu` as listed on the success message, and `ip` is the IP address of your Algo server. If you find yourself regularly logging into the server then it will be useful to load your Algo ssh key automatically. Add the following snippet to the bottom of `~/.bash_profile` to add it to your shell environment permanently.
 
  `ssh-add ~/.ssh/algo > /dev/null 2>&1`
```
Comment (jackivanov) [inline README.md]: Needs to be changed to `algo-path/configs/algo.pem` as well

**Comment by TC1977** [comment]
@jackivanov Would this change be good? It copies the language from the install instructions up top.

**Comment by TC1977** [comment]
@jackivanov I'm realizing that [the setup-roles doc](https://github.com/trailofbits/algo/blob/master/docs/setup-roles.md) largely repeats what's listed on [the deploy-from-ansible doc](https://github.com/trailofbits/algo/blob/master/docs/deploy-from-ansible.md), only with more verbiage. Should these be combined? Or should I update the deploy-from-ansible doc as well (change VPN to Strongswan)?

**Comment by jackivanov** [comment]
I think the best way is to combine it

**Comment by TC1977** [comment]
Ok @jackivanov, done.

**Pull request merged**